core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -872,6 +882,36 @@ protected void doRun() throws Exception {
     }
 
     /**
+     * Get the Jenkins {@link StartupType startup type}.   
+     * @return The Jenkins {@link StartupType startup type}.
+     * @since FIXME
```

Comment:
`@since` is pointless (indeed misleading) on a `@Restricted` method. If for some reason it ever needs to be unrestricted in the future, you would add `@since` for _that_ version.